.papr.yml

```diff
@@ -10,6 +10,8 @@ host:
 
 required: true
 
+timeout: 20
```

Comment:
This is throwing off the parser (need to strengthen that...). It's expecting a number followed by one of s, m, or h. See https://github.com/projectatomic/papr/blob/master/docs/sample.papr.yml#L172.